### PR TITLE
Allow overriding InfoWindow view

### PR DESCRIPTION
**Pull request by josepjaume**
This allows a marker' view overriding its `infoWindow` class.

### CODE REVIEW

**Comment by josepjaume** [comment]
Ping! :panda_face:

**Comment by huafu** [comment]
Oups, sorry! Thanks for the contribution, creating and publishing a release now...

**Comment by huafu** [comment]
Arg, I've merged without checking lol. Needs to define that property's default value on the component!

**Comment by huafu** [comment]
@josepjaume can you re-open a PR, where you'd have fixed what I commented on the diff please?

File: app/templates/components/google-map.hbs [hunk 1]
```diff
@@ -4,7 +4,7 @@
     {{#each marker in _markers itemViewClass=markerViewClass}}
       <li>{{marker.title}} @ {{marker.lat}},{{marker.lng}}</li>
       {{#if view.hasInfoWindow}}
-        {{view 'google-map/info-window' context=marker}}
+        {{view view.infoWindowViewClass context=marker}}
```
Comment (huafu) [inline app/templates/components/google-map.hbs]: `view` isn't necessary in a component, so you can remove it

Comment (huafu) [inline app/templates/components/google-map.hbs]: Also rename `infoWindowViewClass` into `markerInfoWindowViewClass` as it is the one used for markers only, we need to keep them separated

File: app/views/google-map/marker.js [hunk 2]
```diff
@@ -47,6 +47,10 @@ export default GoogleMapCoreView.extend({
     return this.get('controller.infoWindowTemplateName') || this.get('parentView.markerInfoWindowTemplateName');
   }).readOnly(),
 
+  infoWindowViewClass: computed('controller.infoWindowViewClass', 'parentView.infoWindowViewClass', function () {
+    return this.get('controller.infoWindowViewClass') || this.get('parentView.infoWindowViewClass');
+  }).readOnly(),
```
Comment (huafu) [inline app/views/google-map/marker.js]: As explained up, `parentView.infoWindowViewClass` has to be renamed into `parentView.markerInfoWindowViewClass`

File: app/templates/components/google-map.hbs (hunk 1 above)
Comment (huafu) [inline app/templates/components/google-map.hbs]: Ah, you also need to define in the component `markerInfoWindowViewClass` to `'google-map/info-window'` by default ;-)

**Pull request merged**